Feat let cookie name be changed (#885)
teamhanko/hanko: frontend/frontend-sdk/README.md

</script>
```

### Options

You can pass certain options, when creating a new `Hanko` instance:

```js
const defaultOptions = {

>   timeout: 13000,      // The timeout (in ms) for the HTTP requests.
>   cookieName: "hanko", // The cookie name under which the session token is set.
>   storageKey: "hanko"  // The prefix / name of the localStorage keys.

Reviewer: ...and the SDK readme needs an adjustment:
```suggestion
  timeout: 13000,           // The timeout (in ms) for the HTTP requests.
  cookieName: "hanko",      // The cookie name under which the session token is set.
  localStorageKey: "hanko"  // The prefix / name of the localStorage keys.
```

</script>
```

> ### Options
> 
> There are optional options that can be provided when creating a new `Hanko` instance.
> 
> - `timeout` - The timeout (in ms) for the http requests. Default: 13000
> - `cookieName` - The cookie name under which the session token is set. Default: "hanko"
> - `storageKey` - The prefix / name of the localstorage keys. Default: "hanko"

Reviewer: To make it copy & pastable I would suggest the following:
```suggestion
You can pass certain options, when creating a new `Hanko` instance:

```js
const defaultOptions = {
    timeout: 13000,      // The timeout (in ms) for the HTTP requests.
    cookieName: "hanko", // The cookie name under which the session token is set.
    storageKey: "hanko"  // The prefix / name of the localStorage keys.
};

const hanko = new Hanko("http://localhost:3000", defaultOptions);
```
```

> 
> #### Example
> 
> ```typescript
> import { Hanko } from "@teamhanko/hanko-frontend-sdk"
> 
> const hanko = new Hanko("http://localhost:3000", {
>   cookieName: "my-session-cookie"
> })

Reviewer: See comment above
```suggestion
```
